src/components/scenes/newAccount/NewAccountTosScene.tsx

```diff
@@ -225,6 +226,11 @@ export const NewAccountTosScene = (props: NewAccountTosProps) => {
         }
       })
     } catch (e: any) {
+      showError(e)
+      dispatch({
+        type: 'NAVIGATE',
+        data: { name: 'newAccountTos', params: route.params }
+      })
```

Comment:
This looks like a way to trigger a "refresh" of the component, but why not just throw the error to stop the spinner?

Reply:
I might not understand the workflow, so maybe a quick chat could resolve this one.

Reply:
No, at this point we are already on the full screen spinner scene, so this actually goes back to where we came from. Otherwise the user gets stuck on the spinner.